test/create_game_section/new_game1/NewGameViewModel1_unit_test.dart

```diff
@@ -0,0 +1,20 @@
+// Import the test package and Counter class
+import 'package:niira/services/game_service.dart';
```

Comment:
very unneccessary unit test, but it helped me understand how we are using the newgameViewmodel in gameservice. 
then thought its not bad as a smoke test